### DISCUSSION TITLE
API Returning wrong values.

### DISCUSSION THREAD

**Discussion by uhki** [HypixelDev/PublicAPI#497] [Q&A]
I've been trying to use the API to make a bridge stats application, but the API is returning wrong values compared to the stats in-game, I've heard that this problem is happening for everyone, but I've tested on one player.

Tested on player "oNetty"

**Comment by Picsou993** [comment]
I don't have this issue and I don't see why this would happen as the data provided is also the one used in game.
Can you explain more the issue with the affected fields, etc?

**Comment by uhki** [comment]
It is with winstreak, and best winstreak.

**Comment by ConnorLinfoot** [comment]
We would need more information to assist you such as the exact path to the field you are using, but the API is just what is from the database. So it could be the case of using an old field that is no longer supported.

**Reply by uhki** [reply]
path is **player.stats.Duels.current_bridge_winstreak**, note that **.current_winstreak_mode_bridge_duel** is giving the same value as well, just thought i'd point that out

**Reply by mdashlw** [reply]
Could this be related to the fact that the Duels winstreak achievement is currently broken?

**Reply by ConnorLinfoot** [reply]
Winstreaks for duels are no longer stored in the player data to my knowledge, they were moved to a different system in the last update.

**Reply by uhki** [reply]
where exactly?

**Reply by TheNullicorn** [reply]
I believe the API already communicates with other systems to deliver certain info (statuses and leaderboard if I remember).

Would it be too difficult—or worth the higher response time—to add that info to the player endpoint? I imagine win-streaks are be a pretty popular stat.

**Reply by ConnorLinfoot** [reply]
There isn't plans to add this to the API at this time

**Reply by uhki** [reply]
so is it impossible for me to get the winstreak of a player in bridge duels using the API?
